src/main/java/io/logz/jmx2graphite/Jmx2GraphiteJavaAgent.java

```diff
@@ -16,6 +16,8 @@
 public class Jmx2GraphiteJavaAgent {
 
     private static final Logger logger = LoggerFactory.getLogger(Jmx2GraphiteJavaAgent.class);
+    public static final String WHITE_LIST_REGEX = "white-list-regex";
+    public static final String BLACK_LIST_REGEX = "black-list-regex";
```

Comment:
Why have you decided to put this const in public  and also not inline? Using it?